agent/queryparser/helpers.go

```diff
@@ -198,49 +116,39 @@ func parseKeyValueFromComment(s string) (map[string]bool, error) {
 	return res, nil
 }
 
-func prepareMultilineRegexp() error {
+func prepareMySQLComments() error {
 	// to compile regexp only once
-	multilineOnce.Do(func() {
-		multilineRegexp, errMultiline = regexp.Compile(`(?s)\/\*(.*?)\*\/`) //nolint:gocritic
+	mySQLQuotedOnce.Do(func() {
```

Comment:
I wonder if we can simplify the code by using `regexp.MustCompile` to initialize `mySQLQuotedRegexp` (and other regex variables) as package-level variables. This would eliminate the need for `sync.Once` and error handling during regex compilation, since `MustCompile` will panic if the pattern is invalid.

Reply:
Then those `prepare...` functions would not be necessary :)